pytorch_lightning/trainer/states.py

```diff
@@ -58,7 +58,7 @@ class RunningStage(LightningEnum):
     """
     TRAINING = 'train'
     SANITY_CHECKING = 'sanity_check'
-    VALIDATING = 'validation'
+    VALIDATING = 'validate'
```

Comment:
I thought this is used in many places to select the validation dataloader etc, like
method = f"{stage}_dataloader"
Are you sure this is ok?

Reply:
Just double checked, the only place we do that is in the config validator, but there we pass the prefixes manually since `val` is used instead

https://github.com/PyTorchLightning/pytorch-lightning/blob/8fab50f035c328ead9e7f73b7e2f35ab3cfd5451/pytorch_lightning/trainer/configuration_validator.py#L39-L40

https://github.com/PyTorchLightning/pytorch-lightning/blob/8fab50f035c328ead9e7f73b7e2f35ab3cfd5451/pytorch_lightning/trainer/configuration_validator.py#L92-L93

TBH I think I made this change but ended up not relying on it so should be ok to undo it. But I think `validate` fits better with the rest of strings.